### PR TITLE
Apple ball patch 2

### PR DESCRIPTION
**Pull request by apple-ball**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1 +1,8 @@
-Hello World
+<html>
+  <head>
+    <title>Home Page</title>
+  </head>
+  <body>
+    <h1>Hello WOrld!</h1>
```
Comment (Bishal-Khatri) [inline index.html:6]: letter o in world must be small

Comment (Bishal-Khatri) [inline index.html:6]: letter o in world must be small

**Review by Bishal-Khatri — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (apple-ball) [reply index.html:6]: ok

**Review by Bishal-Khatri — APPROVED** [review]

**Review by Bishal-Khatri — APPROVED** [review]
Done.

**Pull request merged**